client/src/Components/UpdateForm/index.js

```diff
@@ -45,8 +77,8 @@ class UpdateForm extends Component {
     } = this.state
     return (
       <div className="updateForm">
-        <form className="upForm">
-          <label class="name" for="name">
+        <form onSubmit={this.updateFormInfo} className="upForm">
+          <label className="name" for="name">
```

Comment:
there's no "for" attribute in react for the label tag, you can use "htmlFor" instead of it